src/main/java/org/zalando/planb/revocation/config/RevocationConfig.java

```diff
@@ -19,7 +24,15 @@
 
     @Bean
     public MessageHasher messageHasher() throws NoSuchAlgorithmException {
-        return new MessageHasher(hashingProperties.getAlgorithms(), hashingProperties.getSalt(),
-                hashingProperties.getSeparator());
+        Map<RevocationType, MessageDigest> hashers = new HashMap<>(hashingProperties.getAlgorithms().size());
```

Comment:
can be final. I would even consider using java 8 and toMap collector: https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html#toMap-java.util.function.Function-java.util.function.Function-